app/parse.py

```diff
@@ -14,5 +20,55 @@ class Course:
     course_type: CourseType
 
 
+def parse_all_courses_info() -> tuple[list[str], list[str]]:
+    page = requests.get(BASE_URL).text
+    soup = BeautifulSoup(page, "html.parser")
+    all_names_of_courses = [
+        name.text
+        for name in soup.find_all(class_="typography_landingH3__vTjok")
+    ]
+    full_time_courses = [
+        course.text
+        for course in soup.find_all(class_="cell large-6 large-offset-1 mb-32")
+    ]
+    part_time_courses = [
+        course.text
+        for course in soup.find_all(class_="cell large-6 large-offset-1")
+    ]
+    all_courses_list = full_time_courses + part_time_courses
+    all_courses = " ".join(all_courses_list).split("Детальніше")
+    all_courses.remove("")
+
+    return all_names_of_courses, all_courses
+
+
+all_names_courses, all_courses_info = parse_all_courses_info()
+
+
 def get_all_courses() -> list[Course]:
-    pass
+    all_courses = []
+
+    for i in range(len(all_names_courses)):
```

Comment:
Can't you just iterate through values, not indexes?

Reply:
I don't care to do this because I apply the index to different lists

Reply:
Okay, I didn't notice the all_courses_info